tests/ui/static/issue-18118-2.rs

```diff
@@ -1,6 +1,9 @@
 pub fn main() {
-    const z: &'static isize = {
-        static p: isize = 3;
-        &p //~ ERROR referencing statics
+    const Z: &'static isize = {
+        //~^ ERROR: it is undefined behavior to use this value
+        static mut P: isize = 3;
```

Comment:
I believe we can fix the test issue by eschewing isize for a fixed-size integer:
```suggestion
    const Z: &'static i32 = {
        //~^ ERROR: it is undefined behavior to use this value
        static mut P: i32 = 3;
```

Reply:
The test is dropped.